_episodes/01-shell-basics.md

```diff
@@ -107,18 +107,18 @@ failed interim execution.
 ## Commonly used shells and their relevance to existing neuroimaging projects
 
 
-- **sh** - a POSIX compliant shell. Generic name, not a specific project
-   - Most portable (since standard)
-   - Many FSL scripts (attempt to be)
-- **ksh** - KornSHell. Based on older bash, but also became a root for tcsh, zsh and others
+- **sh** - a POSIX compliant shell. This is a generic name and doesn’t refer to a specific project.
+   - Most portable (since it is the standard)
+   - Has many FSL scripts
```

Comment:
what I wanted to say was `Many FSL scripts use **sh**` (not that it sh somehow has FSL scripts).  FSL is a software package in neuroimaging